R/condition_message_linter.R

```diff
@@ -1,9 +1,36 @@
 #' Block usage of `paste()` and `paste0()` with messaging functions using `...`
 #'
-#' `stop(paste0(...))` is strictly redundant -- `stop(...)` is equivalent.
-#'   `stop(...)` is also preferable to `stop(paste(...))`. The same applies to
-#'   all default condition functions, i.e., [stop()], [warning()], [message()],
-#'   and [packageStartupMessage()].
+#' Both `stop(paste0(...))` and `stop(paste(...))` are strictly redundant -- `stop(...)` is
```

Comment:
Don't like the rewording... the point is that `stop(paste0(...))` is _exactly_ equivalent to `stop(...)`, but it's not quite so for `stop(paste(...))` because we have to add extra whitespace to account for `sep = " "`, so "strictly redundant" is too strong.

Maybe reword with that in mind?

Reply:
I have reworded this, but not sure if I have stuck the landing with wording. Happy to give it another try.